libr/bin/format/elf/elf.c

```diff
@@ -26,42 +26,47 @@
 #define R_ELF_PART_RELRO 1
 #define R_ELF_FULL_RELRO 2
 
-#define bprintf if(bin->verbose) R_LOG_WARN
+#define bprintf           \
+	if (bin->verbose) \
+	R_LOG_WARN
 
 #define MAX_REL_RELA_SZ (sizeof (Elf_(Rel)) > sizeof (Elf_(Rela))? sizeof (Elf_(Rel)): sizeof (Elf_(Rela)))
 
-#define READ8(x, i) r_read_ble8((x) + (i)); (i) += 1
-#define READ16(x, i) r_read_ble16((x) + (i), bin->endian); (i) += 2
-#define READ32(x, i) r_read_ble32((x) + (i), bin->endian); (i) += 4
-#define READ64(x, i) r_read_ble64((x) + (i), bin->endian); (i) += 8
+#define READ8(x, i)              \
```

Comment:
don't do all this style fixes here please... if you want to clean the file do a separate PR with just whitespace/style fixes, otherwise it is hard for me to review what you did really change.

Reply:
Please see this before I review.

Reply:
You want to revert this change?

Reply:
If possible, yes. There are already many changes and I don't want to risk overlooking something because of those style fixes.

Reply:
> If possible, yes. There are already many changes and I don't want to risk overlooking something because of those style fixes.
I fixed some macro, i you see other style fixes i will try to fix them fast